components/Select/Select.stories.tsx

```diff
@@ -1,9 +1,10 @@
 import React from 'react';
 import { ComponentStory, ComponentMeta } from '@storybook/react';
 
-import { BaseSelect, SelectProps, SelectVariants } from './Select';
+import { Select, SelectProps, SelectVariants } from './Select';
 import { modifyVariantsForStory } from '../../utils/modifyVariantsForStory';
 
+const BaseSelect = (props: SelectProps): JSX.Element => <Select {...props} />;
 const SelectForStory = modifyVariantsForStory<
   SelectVariants,
   SelectProps & React.InputHTMLAttributes<any>
```

Comment:
I don't know if it's expected, select `ReadOnly` story allows opening dropdown and has no different behaviour (styling + interaction) than `Basic` story:


https://user-images.githubusercontent.com/3367393/140497094-5f2c38c6-8bdb-471c-9024-bb36a8bca52f.mp4


From a storybook user perspective, I'd say the component is broken.


Maybe we could comment this story until it's fixed ?